packages/hoppscotch-cli/src/types/request.ts

```diff
@@ -9,6 +10,32 @@ export type FormDataEntry = {
 
 export type HoppEnvPair = { key: string; value: string };
 
+export const HoppEnvKeyPairObject = z.record(z.string(), z.string());
+
+// Shape of the single environment export object that is exported from the app.
+export const HoppEnvExportObject = z.object({
+  name: z.string(),
+  variables: z.array(
+    z.object({
+      key: z.string(),
+      value: z.string(),
+    })
+  ),
+});
+
+// Shape of the bulk environment export object that is exported from the app.
+export const HoppBulkEnvExportObject = z.array(
+  z.object({
+    name: z.string(),
+    variables: z.array(
+      z.object({
+        key: z.string(),
+        value: z.string(),
+      })
+    ),
+  })
```

Comment:
```suggestion
  HoppEnvExportObject
```

We can reuse the above definition, right?